Sources/Afluent/Subscription/AnyCancellable.swift

```diff
@@ -50,3 +50,36 @@ extension AsynchronousUnitOfWork {
         return AnyCancellable(self)
     }
 }
+
+extension AsyncSequence {
+    /// Executes the current async sequence and returns an AnyCancellable token to cancel the subscription.
+    ///
+    /// - Parameters:
+    ///   - receiveCompletion: A function that is executed when the stream has completed normally with `nil` or an error.
+    ///   - receiveOutput: A function that is executed when output is received from the sequence.
+    ///   If this function throws an error, then the stream is completed.
+    public func subscribe(receiveCompletion: ((AsyncSequences.Completion<Error>) async -> Void)? = nil,
+                          receiveOutput: ((Element) async throws -> Void)? = nil) -> AnyCancellable {
+        DeferredTask {
```

Comment:
This is honestly kind of genius. I thought we might have to change the `AnyCancellable` type, but nope!

Reply:
I only realized it because this is basically how I was using Afluent + AsyncSequence. I was like, "this could just be an operator".